sqlalchemy_cockroachdb/requirements.py

```diff
@@ -165,9 +165,9 @@ def sync_driver(self):
     def array_type(self):
         # DDL like
         #
-        # CREATE TABLE foo (thing INTEGER[])
+        # CREATE TABLE foo (thing INTEGER[][])
         #
-        # throws 'FeatureNotSupported: at or near "]": syntax error: unimplemented'
+        # throws 'invalid syntax: statement ignored: at or near "]": syntax error: unimplemented'
```

Comment:
ah that's right, CRDB doesn't have multi-dimensional array support (https://github.com/cockroachdb/cockroach/issues/32552). does this exclusion only cover those, or is it all arrays?

Reply:
It is all arrays at the moment, because the test uses a [fixture](https://github.com/sqlalchemy/sqlalchemy/blob/756998c099ef3c7e1e52c021088c634f256b7439/lib/sqlalchemy/testing/suite/test_types.py#L237) that creates both types of columns (single_dim and multi_dim) in the same table.